velox/functions/prestosql/aggregates/MinMaxByAggregates.cpp

```diff
@@ -1175,7 +1175,7 @@ exec::AggregateRegistrationResult registerMinMaxBy(const std::string& name) {
           if (isRawInput) {
             // Input is: V, C.
             return create<Aggregate, Comparator, isMaxFunc>(
-                resultType, argTypes[0], argTypes[1], errorMessage);
+                resultType, argTypes[0], argTypes[1], errorMessage, true);
```

Comment:
We need to pass throw-on-nested-nulls flag in the 'else' branch as well. That's because an Aggregate function instance may receive a mix of raw inputs and intermediate results (which happens in streaming case).

Reply:
Fixed

Reply:
With the recent changes to address https://github.com/facebookincubator/velox/issues/6506, isRawInput is always true. Hence, we can simplify this code by removing the else branch.